### DISCUSSION TITLE
Links to notebooks in pdf (quarto manuscript)

### DISCUSSION THREAD

**Discussion by udurraniAtPresage** [quarto-dev/quarto-cli#9996] [Q&A] [answered]
### Description

I am trying out the quarto manuscript format using the [example template repo](https://github.com/quarto-ext/manuscript-template-rstudio/tree/main) from the documentation. Upon rendering the `index.qmd`, I see an html page with source notebook links, e.g., `Source: Article Notebook`. I see these links in the generated pdf file also. Two questions:   

1. Clicking on the these links on the html page take me to the relevant section in the notebook. But doing the same in the pdf tries to visit `https://quarto-ext.github.io/manuscript-template-rstudio/index.qmd.html`. Is this a bug?
2. How to turn off the links in pdf but keep them in the html file?

**Comment by mcanouil** [comment, marked as answer]
The link is built from the git remote information I believe, so since you probably clone locally the repository, I would say that's the expected result.

You should be able to deactivate links following the documentation: https://quarto.org/docs/output-formats/html-multi-format.html

**Reply by udurraniAtPresage** [reply]
Thank you.
